miniupnpd/INSTALL

```diff
@@ -173,9 +173,9 @@ allowed, so it is a good practice to have a "catch all" deny permission
 rule at the end of your permission ruleset.
 Sample permission ruleset :
 allow 4662-4672 192.168.1.34/32 4662-4672
-deny 0-65535 192.168.1.34/32 0-65535
+deny 1-65535 192.168.1.34/32 1-65535
 allow 1024-65535 192.168.1.0/24 1024-65535
-deny 0-65535 0.0.0.0/0 0-65535
+deny 1-65535 0.0.0.0/0 1-65535
```

Comment:
whats is the point of this change ?

Reply:
:facepalm: the BSD sockets use port 0 for "any" port.
your modification is adding confusion.

The purpose of the "catch all" deny rule is to deny everything. Allowing request for port 0 is a big risk of mix-up.